src/openrct2/paint/tile_element/Paint.LargeScenery.cpp

```diff
@@ -301,8 +301,7 @@ void large_scenery_paint(paint_session* session, uint8_t direction, uint16_t hei
         }
         // 6B8331:
         // Draw sign text:
-        set_format_arg(0, uint32_t, 0);
-        set_format_arg(4, uint32_t, 0);
+        Formatter::Common().Add<uint32_t>(0).Add<uint32_t>(0);
```

Comment:
Ditto

Reply:
I wouldn't be able to tell. I don't know if the arguments are cleared for the same reason.